pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,248 @@
+    "- A blank user group for the `$OWNER_TEAM` if it does not exist yet in the `cluster-scope/base/user.openshift.io/groups/$OWNER_TEAM`\n",
+    "- An RBAC component for the project admin role `RoleBinding` in `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM` and maps it to the newly added namespace.\n",
+    "\n",
+    "##### Enabling namespace deployment to Zero cluster\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!cd cluster-scope/overlays/prod/$ENV/$TARGET_CLUSTER"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Open the `kustomization.yaml` file in this folder and add `../../base/core/namespaces/$NAMESPACE` to the `resources` field. We aim to keep this field alphabetically sorted for better human readability:\n",
+    "```\n",
+    "---\n",
+    "apiVersion: kustomize.config.k8s.io/v1beta1\n",
+    "kind: Kustomization\n",
+    "\n",
+    "resources:\n",
+    "...\n",
+    "- ../../base/core/namespaces/$NAMESPACE\n",
+    "...\n",
+    "```\n",
+    "##### Authenticate via OpenShift\n",
+    "Users are automatically created upon login to the appropriate cluster. This is done by logging in via SSO (any gmail account will work).\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "For the `MOC/Zero` you can access the login from this [link](https://console-openshift-console.apps.zero.massopen.cloud/).\n",
+    "This will lead the user to an authentication provider selection screen:\n",
+    "\n",
+    "```{image} ../img/openshift-login.png\n",
+    ":alt: openshift-login\n",
+    ":align: center\n",
+    "```\n",
+    "<br/><br/>\n",
+    "Please select `operate-first` provider.\\\n",
+    "With the user now created, we will need to provide them with appropriate rbac access to this namespace.\n",
+    "\n",
+    "#### Giving Users Rbac Permissions\n",
+    "The following steps enable users to access designated cluster/namespaces. This consists of adding users to the appropriate OpenShift groups.\n",
+    "\n",
+    "Navigate to `cluster-scope/overlays/prod/$ENV/$TARGET_CLUSTER`.\n",
+    "```{note}\n",
+    "For moc/zero or moc/infra navigate to cluster-scope/overlays/prod/moc/common instead\n",
+    "```"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!cd cluster-scope/overlays/prod/$ENV/$TARGET_CLUSTER"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Create the following resource describing the users in your group:\n",
+    "\n",
+    "```\n",
+    "# groups/GROUP_NAME.yaml\n",
+    "apiVersion: user.openshift.io/v1\n",
+    "kind: Group\n",
+    "metadata:\n",
+    "  name: GROUP_NAME\n",
+    "  annotations:\n",
+    "    kustomize.config.k8s.io/behavior: replace\n",
+    "users:\n",
+    "  - USER_1\n",
+    "  - USER_2\n",
+    "```\n",
+    "```{note}\n",
```

Comment:
Instead of this point users to the group resource created by `opfcli` in `cluster-scope/base/user.openshift.io/groups/<name>/group.yaml` and make them add users in there.

Also mention that the usernames are **GitHub usernames, converted to lowercase**

Reply:
Resolved in latest commit.